src/components/buttons/ButtonSolid.tsx

```diff
@@ -265,7 +265,10 @@ export const ButtonSolid = React.forwardRef<View, ButtonSolidProps>(
         // Using || operator because empty string is not an accepted value
         accessibilityLabel={accessibilityLabel || label}
         accessibilityHint={accessibilityHint}
-        accessibilityState={{ busy: loading }}
+        accessibilityState={{
+          busy: loading,
+          disabled: disabled || false
```

Comment:
```suggestion
          disabled
```

Reply:
It doesn't work because we pick the `disabled` prop from Pressable and it could be also undefined. Look at the last commit where I fixed the typescript error